frontend/src/app/reducers/addon.js

```diff
@@ -84,8 +84,7 @@ function setHasAddon(
   } catch (e) {}
   return {
     ...state,
-    hasAddon,
-    installed: hasAddon ? state.installed : {}
```

Comment:
we no longer want to wipe the installed experiment list when the txp addon is removed